src/entities/Game.entity.ts

```diff
@@ -146,15 +165,17 @@ export class Game extends BaseEntity {
   description: string;
 
   @ApiProperty({
-    description: 'The community type of this game',
+    enum: Community,
+    default: Community.DISQUS,
   })
   @Column({ default: Community.DISQUS })
   @IsEnum(Community)
   @IsNotEmpty()
   community: Community;
 
   @ApiProperty({
-    description: 'The category that best describes this game',
```

Comment:
Why did you remove the description？

Reply:
Yeah, it was by accident. I've copied them back.